language/move-prover/tests/sources/stdlib/modules/libra_account.move

```diff
@@ -442,6 +447,28 @@ module LibraAccount {
         account: Self::T,
         addr: address,
     );
+    spec fun save_account { // Prover does not attempt to prove this spec because it's a native function.
```

Comment:
It may not be an urgent thing, but it would be great if Prover automatically proves this spec against the builtin Boogie model of this native function, rather than having an extra dummy function (`verify_save_account` below) to do that.

Reply:
Are you asking about a general feature in Move Prover or something specific to this example?

Reply:
Yeah, I meant a general feature. But, it's a matter of convenience, and may not need be high-priority right now.

Reply:
This is rather easy to implement. We just need to generate the _verify function for a native function, while still skipping the inlined function.